portfolio/src/main/webapp/index.html

```diff
@@ -31,140 +35,176 @@ <h1>About Me</h1>
               artifical intelligence and machine learning, as well as the interesction of coding and hardware. In my free 
               time I enjoy playing basketball (Lakers fan), reading, and watching Netflix.
             </p>
-            <h2 class="details"><span class="first">School:</span> Massachusetts Institue of Techonology</h2>
-            <h2 class="details"><span class="first">Major:</span> Computer Science & Electrical Engineering</h2>
-            <h2 class="details"><span class="first">Location:</span> Las Vegas, Nevada</h2>
+            <h2 class="details"><span class="first">School:</span>Massachusetts Institue of Techonology</h2>
+            <h2 class="details"><span class="first">Major:</span>Computer Science & Electrical Engineering</h2>
+            <h2 class="details"><span class="first">Location:</span>Las Vegas, Nevada</h2>
           </div> 
+        </div>
+    </section> 
+    </div> 
+ 
+    <div class="resume">
+        <section>
+        <h1>Skills & Experience</h1>
+        <hr class="short">
+        <h2>Education</h2>
+        <div class"resume-container">
+            <div class="section">
+                <div class="header">
+                    <h3>Massachusetts Institue of Techonology</h3>
+                    <h4>Candidate for BS in EECS</h3>
+                    <p>Cambridge,MA<p>
```

Comment:
This isn't a code nit but just a question: I'm noticing that in some places you have a space after a comma in the text and in some places you don't, is this intentional?

Reply:
That is just me being inconsistent, I will fix them.